projects/scion/workbench/src/lib/spec/view-part-grid.component.spec.ts

```diff
@@ -384,15 +384,15 @@ describe('ViewPartGridComponent', () => {
 
     // Add View 1
     wbRouter.navigate(['view-1'], {blankViewPartRef: 'viewpart.1'}).then();
-    advance(fixture);
+    advance(fixture, 1);  // Wait 1ms before adding view2, otherwise the touchTimestamp of both views will be the same
```

Comment:
Is there no better way?
Maybe it would be clearer to not set `Date.now` timestamp in `InternalWorkbenchView.activate` directly but retrieving the timestamp from a service instead (e.g. ViewActivateTimestampProvider). In our tests, we could provide a dummy service which returns an increment. This has the advantage that our tests do not have to simulate the asynchronous passage of time to make view activation work as expected.

Reply:
done